hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -1295,6 +1337,42 @@ public final HoodieTable initTable(WriteOperationType operationType, Option<Stri
     return table;
   }
 
+  public void validateAgainstTableProperties(HoodieTableConfig tableConfig, HoodieWriteConfig writeConfig) {
```

Comment:
this has been moved from HoodieTableMetaClient.. this was only called here. and we should not be validating write configs at that layer.